examples/cpp/pyperf/PyPerfCollapsedPrinter.cc

```diff
@@ -97,6 +98,12 @@ void PyPerfCollapsedPrinter::processSamples(
           lostSymbols++;
         }
       }
+      // TODO check 0 >
```

Comment:
I get `-14 EFAULT` sometimes, I think it's for "no kernel".
I also get `-17 EEXIST` when the hash already exists, need to figure out how to handle.